commerce-test-util/src/main/java/com/liferay/commerce/test/util/TestCommerceContext.java

```diff
@@ -29,6 +29,19 @@
  */
 public class TestCommerceContext implements CommerceContext {
 
+	public TestCommerceContext(
+		CommerceCurrency commerceCurrency, CommerceChannel commerceChannel,
+		User contextUser, Group contextGroup, CommerceAccount commerceAccount,
+		CommerceOrder commerceOrder) {
+
+		_commerceCurrency = commerceCurrency;
+		_commerceChannel = commerceChannel;
+		_contextUser = contextUser;
+		_contextGroup = contextGroup;
+		_commerceAccount = commerceAccount;
+		_commerceOrder = commerceOrder;
+	}
+
 	public TestCommerceContext(
```

Comment:
Since you've now added a constructor with more params, can you have this old constructor call the new one (with `null` passed in)? I know right now there's literally no logic to share between them, but it leaves less room for bugs to be introduced in the future.